internal/cephfs/controllerserver.go

```diff
@@ -701,7 +712,7 @@ func (cs *ControllerServer) CreateSnapshot(
 			}
 		}
 	}()
-	snap, err := doSnapshot(ctx, parentVolOptions, sID.FsSnapshotName)
+	snap, err := doSnapshot(ctx, req, parentVolOptions, sID.FsSnapshotName)
```

Comment:
pass only `k8s.GetSnapshotMetadata(parameters)` for from here. we dont need to pass whole `req`